develop/devguide/Automation/UIBlockTypesOverview.md

```diff
@@ -6,23 +6,44 @@ uid: UIBlockTypesOverview
 
 The [UIBlockType](xref:Skyline.DataMiner.Automation.UIBlockType) enum defines different types of UI components.
 
-|Name  |Description  |
-|---------|---------|
-|Button     |Button.         |
-|Calendar     |Calender control.         |
-|CheckBox     |Checkbox.         |
-|CheckBoxList     |Checkbox list.         |
-|DropDown     |Drop-down list.         |
-|Executable     |Executable.         |
-|FileSelector     |File selector.         |
-|Numeric     |Numeric.         |
-|Parameter     |Text displaying the value of a parameter.         |
-|PasswordBox     |Password input box.         |
-|RadioButtonList     |Radio button list.         |
-|StaticText     |Static text.         |
-|TextBox     |Text box.         |
-|Time     |Item that displays a time value.         |
-|TreeView     |Tree view control.         |
+| Name | Description |
+|---|---|
+| [Button](#button) | Button. |
+| [Calendar](#calendar) | Calendar control. |
+| [CheckBox](#checkbox) | Checkbox. |
+| [CheckBoxList](#checkboxlist) | Checkbox list. |
+| [DropDown](#dropdown) | Dropdown list. |
+| [Executable](#executable) | Executable. |
+| [FileSelector](#fileselector) | File selector. |
+| [Numeric](#numeric) | Numeric. |
+| [Parameter](#parameter) | Text displaying the value of a parameter. |
+| [PasswordBox](#passwordbox) | Password input box. |
+| [RadioButtonList](#radiobuttonlist) | Radio button list. |
+| [StaticText](#statictext) | Static text. |
+| [TextBox](#textbox) | Text box. |
+| [Time](#time) | Item that displays a time value. |
+| [TreeView](#treeview) | Tree view control. |
+
+## UIBuilder
+
+To create these UIBlocks, a UIBuilder should be defined with *Width*, *RowDefs*, and *ColumnDefs*.
+
+```csharp
+var MyDialogBox = new UIBuilder() { Width = 800, RowDefs = "auto;auto", ColumnDefs = "auto;auto" };
```

Comment:
Would change 'MyDialogBox' to 'uiBuilder' and make sure it is referenced everywhere like this. I found at least one (Numeric section) where something else is used.